app/views/granules/prepare_script.html.erb

```diff
@@ -132,16 +133,25 @@ fetch_urls <<'EDSCEOF'
 <script>
   (function () {
 
-    function downloadScriptFile() {
-      var content = document.getElementById("script").textContent;
-      var dl = document.createElement('a');
-      dl.setAttribute('href', 'data:text/csv;charset=utf-8,' + encodeURIComponent(content));
-      dl.setAttribute('download', 'download.sh');
-      dl.click();
+    function downloadFileIE(e) {
+      var blob = new Blob([e.target.scriptFileContent]);
+      window.navigator.msSaveOrOpenBlob(blob, e.target.scriptFileName);
     }
 
     document.addEventListener('DOMContentLoaded', function () {
-      document.getElementById('download-button').addEventListener('click', downloadScriptFile);
+      var dlLink = document.getElementById('download-button');
+      dlLink.scriptFileContent = document.getElementById("script").textContent;
+      dlLink.scriptFileName = 'download.sh';
+
+      var isIE = /*@cc_on!@*/false || !!document.documentMode;
```

Comment:
@cc_on is only for IE. So if it is IE,` /*@cc_on!@*/false` compiles to `!false`.